### PR TITLE
Things list: Add grouping by location & Display location in list entries

### LINKED ISSUE #2099: Grouping things by Loacation
It would be convenient to group tnings not only by type of binding, but also by location
image

### PR DESCRIPTION
**Pull request by d51x**
Closes #2099.

### CODE REVIEW

**Comment by d51x** [comment]
Discussion #2099

**Review by florian-h05 — APPROVED** [review]
LGTM, thanks! Nice addition.

BTW: You do not need to manually request review from those who GitHub suggests ... the maintainer group is automatically requested for review and all maintainers are notified.

**Comment by florian-h05** [comment]
@d51x I have just noticed you created your PR from the wrong branch and to the wrong branch.

Only use `main` for working on the UI code, and only open PRs to `main`. 
I have corrected your PR by doing some Git magic, but please remember to use the `main` branch in the future.

**Pull request merged**
